src/test/java/plugins/LogParserTest.java

```diff
@@ -36,6 +41,83 @@ public void globalConfig() {
     }
 
     /**
+     * Test that the link from the sidebar points to a valid position in the content page and
+     * that the text in the content page has the correct color.
+     */
+    @Test
+    public void testLinksAndColor() throws Exception {
+        Job job = configureSampleJob();
+
+        Build build = job.startBuild().waitUntilFinished();
+        build.open();
+
+        driver.findElement(By.partialLinkText("Parsed Console Output")).click();
+        LogParserOutputPage outputPage = new LogParserOutputPage(build);
+
+        assertThat(outputPage.getFragmentOfContentFrame("Error", 1), is("ERROR1"));
+
+        assertThat(outputPage.getColor("Error",1), is("red"));
+    }
+
+    /**
+     * Test the number of warnings and errors recognized by the plugin.
+     */
+    @Test
+    public void testErrorReporting() throws Exception {
+        Job job = configureSampleJob();
+
+        Build build = job.startBuild().waitUntilFinished();
+
+        build.open();
+
+        // Summary has correct values
```

Comment:
Remove comment and create a method like "checkSummary" with the code that is referred to by the comment. 

(Taken from Clean Code: A Handbook of Agile Software Craftsmanship, see extract at
http://www.kyleblaney.com/software-blog/2012/6/29/comments-are-a-failure.html)